### PR TITLE
Added cart page

### PR DESCRIPTION
**Pull request by aishwarya-bm**
Added cart items & price details.
Responsiveness not handled yet.

https://user-images.githubusercontent.com/66695068/154565476-e32f3a17-410a-41ac-9ddc-f36b8e1ec379.mp4

### CODE REVIEW

File: screens/cart.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../stylesheets/screens/productslist.css">
+    <title>Cart | ArtStation</title>
```
Comment (shuntzhere) [inline screens/cart.html]: Can this second stylesheet be imported to the main style.css?

File: screens/cart.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                <button class="btn btn-link nav-btn">Search</button>
+            </div>
+            <div class="nav-pills d-flex">
+                <ul class="list list-no-bullet children-center">
+                    <li class="list-item">
+                        <a href="../index.html" class="btn btn-link nav-btn ">LOGIN</a>
+                    </li>
+                    <li class="list-item">
+                        <a href="./home.html"><i class="fa fa-solid fa-home"></i></a>
+                    </li>
```
Comment (shuntzhere) [inline screens/cart.html:33]: Your logo already redirects to the home page. You don't need another button for doing the same thing.

File: screens/cart.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        <div class="cart-container d-flex">
+            <div class="cart-items">
+                <h3 class="text-center">My Cart</h3>
+                <ul class="list-no-bullet d-grid grid-gap">
+                    <li>
+                        <div class="card card-hor d-flex">
+                            <img class="card-media"
+                                src="https://i0.wp.com/stationerie.co.in/wp-content/uploads/2021/05/1A3B627D-C304-4070-992B-CA499237F962-scaled.jpeg?resize=600%2C600&ssl=1"
+                                alt="" />
+                            <div class="card-data children-stacked p-rel">
```
Comment (shuntzhere) [inline screens/cart.html]: Don't leave a blank alt.

File: screens/cart.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                            </div>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div class="card card-hor d-flex">
+                            <img class="card-media"
+                                src="https://i0.wp.com/stationerie.co.in/wp-content/uploads/2021/05/1A3B627D-C304-4070-992B-CA499237F962-scaled.jpeg?resize=600%2C600&ssl=1"
+                                alt="" />
+                            <div class="card-data children-stacked p-rel">
```
Comment (shuntzhere) [inline screens/cart.html]: Here as well.

File: screens/cart.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+
+                    <li>
+                        <div class="card card-hor d-flex">
+                            <img class="card-media"
+                                src="https://i0.wp.com/stationerie.co.in/wp-content/uploads/2021/05/1A3B627D-C304-4070-992B-CA499237F962-scaled.jpeg?resize=600%2C600&ssl=1"
+                                alt="" />
```
Comment (shuntzhere) [inline screens/cart.html]: Here's an alt too.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
The page is looking good, Suggested some changes.

File: screens/productslist.html [hunk 6]
```diff
@@ -115,6 +115,19 @@ <h5>Rating</h5>
                                 <label for="oneplus">1 stars & above</label>
                             </li>
                         </ul>
+                        <hr>
+                        <h5>Sort by Price</h5>
+                        <ul class="list list-no-bullet">
+                            <li class="list-item">
+                                <input type="radio" id="lth" name="lth" value="lth" checked>
```
Comment (therajatg) [inline screens/productslist.html:122]: What is checked?

**Review by therajatg — APPROVED** [review]
This looks good

File: stylesheets/screens/cart.css [hunk 7]
```diff
@@ -32,9 +32,6 @@
     background: var(--gold-color);
 }
 
-.cart-items{
-    flex: 0 0 calc(100vw - 400px);
-}
 
 .cart-price {
     flex: 0 0 300px;
```
Comment (therajatg) [inline stylesheets/screens/cart.css:33]: why flex basis in px and not rem

**Review by therajatg — APPROVED** [review]
looks clean

**Review by therajatg — APPROVED** [review]
looks clean

**Review by therajatg — APPROVED** [review]
looks ready to merge

File: stylesheets/screens/cart.css (hunk 7 above)
Comment (aishwarya-bm) [reply stylesheets/screens/cart.css:33]: I wanted an absolute value so I have given in px here

File: screens/cart.html (hunk 5 above)
Comment (aishwarya-bm) [reply screens/cart.html]: fixed

File: screens/cart.html (hunk 4 above)
Comment (aishwarya-bm) [reply screens/cart.html]: fixed

File: screens/cart.html (hunk 3 above)
Comment (aishwarya-bm) [reply screens/cart.html]: fixed

File: screens/cart.html (hunk 2 above)
Comment (aishwarya-bm) [reply screens/cart.html:33]: Yeah but I want to have both

File: screens/cart.html (hunk 1 above)
Comment (aishwarya-bm) [reply screens/cart.html]: made changes to include only one main css

**Review by shuntzhere — APPROVED** [review]
Looks good.

**Pull request merged**